sdcm/tester.py

```diff
@@ -701,6 +701,25 @@ def get_cluster_aws(self, loader_info, db_info, monitor_info):
             else:
                 db_info['device_mappings'] = []
 
+        additional_ebs_volumes_num = self.params.get("aws_ebs_volume_num")
+        if additional_ebs_volumes_num > 0:
```

Comment:
It's not requires to make this check, because loop over `range(0)` (and `range(negative_value)`) will do the same thing for you.

Reply:
Live this logic dur to fixes for comments below

Reply:
@aleksbykov can you clarify ? didn't understood your answer here...

Reply:
@fruch , yes sorry for grammar. Done